src/views/ettersending/Ettersending.tsx

```diff
@@ -115,37 +115,60 @@ export function Ettersending(props: IProps) {
           ))}
         </ErrorSummary>
       )}
-
       {ettersendingText && (
         <div className={styles.dokumentasjonsTextContainer}>
           <PortableText value={ettersendingText.body} />
         </div>
       )}
-
       {availableDokumentkravForEttersending.length > 0 && (
         <>
           <Heading level="2" size="medium" className="my-6">
             {getAppText(ETTERSENDING_DOKUMENTER_INNSENDING_TITTEL)}
           </Heading>
 
-          {availableDokumentkravForEttersending.map((krav) => (
-            <EttersendingDokumentkravSendingItem
-              key={krav.id}
-              dokumentkrav={krav}
-              removeDokumentkrav={removeDokumentkrav}
-              addDokumentkrav={addDokumentkravWithNewFiles}
-              hasBundleError={
-                !dokumentkravWithBundleError.findIndex(
-                  (dokumentkrav) => dokumentkrav.id === krav.id
-                )
-              }
-            />
-          ))}
+          {availableDokumentkravForEttersending.map((krav: IDokumentkrav) => {
```

Comment:
Forslag: Istedet for bruke availableDokumentkravForEttersending to ganger med if inni kan vi heller lage egne array for hver av de. Da blir det mindre logikk i uttegningen.
Noe sånt: 
```
  const dokumentkravMissingFiles = props.dokumentkrav.krav.filter(
    (krav) =>
      (krav.svar === DOKUMENTKRAV_SVAR_SEND_NAA || krav.svar === DOKUMENTKRAV_SVAR_SENDER_SENERE) &&
      !krav.bundleFilsti
  );

  const dokumentkravReceived = props.dokumentkrav.krav.filter(
    (krav) =>
      (krav.svar === DOKUMENTKRAV_SVAR_SEND_NAA || krav.svar === DOKUMENTKRAV_SVAR_SENDER_SENERE) &&
      krav.bundleFilsti
  );
```